docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -328,6 +328,61 @@ When I take screenshot and save it to folder `$path`
 When I take screenshot to path `${screenshot-directory}/#{generate(Ancient.god)}.png`
 ----
 
+=== Scan QR code
+
+Scan a QR code and save its value to the variable with the specified variable
+
+Supported Code Formats
+
+[cols="2,2,2", options="header"]
+|===
+
+|1D product
+|1D industrial
+|2D
+
+a|
+* UPC-A
+* UPC-E
+* EAN-8
+* EAN-13
+* UPC/EAN Extension 2/5
+* RSS-14
+* RSS-Expanded
+a|
+* Code 39
+* Code 93
+* Code 128
+* Codabar
+* ITF
+a|
+* QR Code
```

Comment:
since QR code is the only one of the supported formats, shouldn't we change `QR code` value to something more generic, e.g. `barcode`?